submission/distributor.go

```diff
@@ -298,28 +325,48 @@ func BuildLogClient(log *loglist2.Log) (client.AddLogClient, error) {
 // the local copy of the roots up-to-date.
 func NewDistributor(ll *loglist2.LogList, plc ctpolicy.CTPolicy, lcBuilder LogClientBuilder, mf monitoring.MetricFactory) (*Distributor, error) {
 	var d Distributor
+	// Divide Logs by statuses.
+	d.ll = ll
 	usableStat := make([]loglist2.LogStatus, 1)
 	usableStat[0] = loglist2.UsableLogStatus
 	active := ll.SelectByStatus(usableStat)
-	d.ll = &active
+	d.usableLl = &active
+	pendingQualifiedStat := make([]loglist2.LogStatus, 2)
+	pendingQualifiedStat[0] = loglist2.PendingLogStatus
+	pendingQualifiedStat[1] = loglist2.QualifiedLogStatus
```

Comment:
Couldn't you just do
```
pendingQualifiedStat := []loglist2.LogStatus{
	loglist2.PendingLogStatus,
	loglist2.QualifiedLogStatus,
}
```
instead of the above 3 lines?

(similarly couldn't is also be 
```
usableStat := []loglist2.LogStatus{loglist2.UsableLogStatus}
```
above that)

Reply:
Makes sense